### PR TITLE
refactor create account w/ ledger

### PR DESCRIPTION
**Pull request by Patrick1904**
Create account with Ledger key instead of randomly generated key. Mainly to fix this https://github.com/near/near-wallet/issues/828

### CODE REVIEW

**Comment by kcole16** [comment]
@Patrick1904 @vgrichina @mattlockyer Do we think this is ready to merge as a hotfix?

**Comment by vgrichina** [comment]
@kcole16 not yet. I'll make a pass

File: src/translations/en.global.json [hunk 1]
```diff
@@ -66,6 +66,9 @@
         },
         "deleteRecoveryMethod": {
             "modal": "You will need to confirm delete key details on your Ledger."
+        },
+        "LedgerPublicKey": {
```
Comment (vgrichina) [inline src/translations/en.global.json:70]: @Patrick1904 where is this string used. I wonder if I'm confused or some file is missing from commit.

Comment (Patrick1904) [reply src/translations/en.global.json:70]: @vgrichina this what [`modal.textId`](https://github.com/near/near-wallet/blob/ledger-create-account/src/components/accounts/ledger/LedgerConfirmActionModal.js#L24) returns for the `Translate` `id` in the `LedgerConfirmActionModal` while requesting the Ledger key

**Pull request merged**